build.gradle

```diff
@@ -167,8 +170,35 @@ subprojects {
   }
 }
 
+task codeCoverageReport (type: JacocoReport, group: 'Coverage reports') {
+  description = "Creates an aggregate coverage for the whole project."
+  def reportingProjects = subprojects.findAll()
+  dependsOn(reportingProjects.tasks.collectMany{it.withType(Test)})
+
+  onlyIf = { true } // required to circumvent a wrong check in the jacoco report plugin (v2.13)
```

Comment:
Its a comment in the reporter, I haven't seen an issue. It is probably already fixed in gradle 3.4.x, but that update is not trivial so that's why I haven't included it here.